components/authz-service/server/v2/authz.go

```diff
@@ -27,25 +24,23 @@ import (
 type authzServer struct {
 	log      logger.Logger
 	engine   engine.V2Authorizer
-	vSwitch  *VersionSwitch
 	projects api.ProjectsServer
 	store    storage.Storage
 }
 
 // NewPostgresAuthzServer returns a new IAM v2 Authz server.
-func NewPostgresAuthzServer(l logger.Logger, e engine.V2Authorizer, v *VersionSwitch, p api.ProjectsServer) (api.AuthorizationServer, error) {
+func NewPostgresAuthzServer(l logger.Logger, e engine.V2Authorizer, p api.ProjectsServer) (api.AuthorizationServer, error) {
```

Comment:
no need to version switch-- there will just be one and this was causing problems